### PR TITLE
Update readme

### LINKED ISSUE #642: Add examples in readme

### LINKED ISSUE #918: docs: Explain the difference between using InstallAsync and the dotnet tool

### PR DESCRIPTION
**Pull request by kblok**
closes #642
closes #918

### CODE REVIEW

**Comment by Meir017** [comment]
looks to me like both files are identical.
can we run `cp readme.md docfx_project/index.md` in the docs github action?

**Comment by kblok** [comment]
@Meir017 yeah, I created #921 for that :)

**Review by arjunattam — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,7 +12,98 @@ PlaywrightSharp is a .Net library to automate [Chromium](https://www.chromium.or
...
+
+```cs 
+await Playwright.InstallAsync();
+var playwright = await Playwright.CreateAsync();
+```
+
+`Playwright.InstallAsync()` will download the required browsers. `Playwright.CreateAsync()` will create and launch the playwright driver.
+
+```cs
+await Playwright.InstallAsync();
```
[the remark this answers is not in the document]
Comment (kblok) [reply README.md:26]: @arjunattam my feeling is that the community will make that decision for us.
These are a few thoughts I have related to this feature:
 * Dotnet tools are quite new, and I don't know if developers are used to these kinds of tools. But that's also up to us. If we remove the `InstallAsync` from all our examples, and we explain it in the readme, we would be teaching users to use this tool.
 * I think Playwright-Sharp should work out-of-the-box. I want to add the nuget package, write some code, and get it running without any extra steps.
 * The dotnet tool adds value when Playwright Sharp is used in production apps, where you don't want the app to install browsers in runtime.
 * In unit tests, and even if we create extensions in the future, it won't hurt having the `InstallAsync` somewhere.

[the remark this answers is not in the document]
Comment (arjunattam) [reply README.md:26]: Thanks for sharing. This all makes sense. With the Playwright cli and driver merging together, I think we can come up with a better integration.

I have to remind myself that `InstallAsync` is `DownloadBrowsersIfNeeded` but that might just be me :)

**Pull request merged**